terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/redirect/RedirectTest.java

```diff
@@ -108,20 +124,50 @@ public void test01_03_redirectToExternalLink() {
 
         driver.findElement(By.id("btn1")).click();
 
-        // Unlike other application servers, WebSphere Liberty Profile & WebSphere traditional wraps an unexpected exception of
-        // ServletException.
-        // So the expected error page is different.
-        String expectedErrorMessage;
-        if (apServerName == ApServerName.WEBSPHERELP
-                || apServerName == ApServerName.WEBSPHERETR) {
-            expectedErrorMessage = "System Error...";
+        // check include "/terasoluna-gfw-functionaltest-web" in URL.
+        assertTrue(driver.getCurrentUrl().contains(
```

Comment:
- [ ] You should not use `Assert#assertTrue`, please change to `Assert#assertThat`.

Reply:
0d1f352b4a6998f20173589a5024803d8bcf7604

I changed to use `Assert#assertThat` .